tests/test-mocha.js

```diff
@@ -0,0 +1,5 @@
+// this is the polyfil so node has a crypto.subtle etc.
+const crypto = require('isomorphic-webcrypto');
+global.crypto = crypto;
```

Comment:
We need to figure out the right webpack aliasing/whatever magic so we don't have to use `global`. Seems like we'd want some `crypto.js` file to pull in that, on node, exports `isomorphic-webcrypto` and in the browser exports `window.crypto || window.msCrypto` or something to that effect. Then we'd import `./crypto.js` in the various algorithm files.

cc: @davidlehn

Reply:
oh yeah good idea webpack actually has a boolean that exposes crypto to the bundle. so I will look into that today.

Reply:
@dlongley actually the mocha tests just use the source dir (not even babel is used). we could test on a webpack build though of the source with a polyfill though. webpack already provided a polyfil for node crypto for browsers the issue here is that these tests need subtle which is a browser only crypto library.

Reply:
Right -- we're doing this "browser-first", so we need to polyfill node, not the browser. There is a pattern where you can do aliasing with webpack such that my solution in comment https://github.com/digitalbazaar/data-hub-client/pull/3#discussion_r277301573 should work (I think). @davidlehn -- can you confirm?

Reply:
I believe that webpack plugins https://webpack.js.org/guides/shimming/

would handle this. as webpack the existing setup:

```
node: {
  crypto: true
}
```

actually does the opposite i.e. it puts a node crypto polyfil in the browser.
https://webpack.js.org/configuration/node/

so yeah expand we need this:

```
plugins: [
     new webpack.ProvidePlugin({
       crypto: 'isomorphic-webcrypto'
     })
]
```

@davidlehn confirm?

Reply:
@davidlehn basically make sure that the crypto stuff handled by webpack is being handled correctly here.